bears/general/FilenameBear.py

```diff
@@ -42,6 +43,9 @@ def run(self, filename, file,
             - ``pascal`` (``ThisIsPascalCase``)
             - ``snake`` (``this_is_snake_case``)
             - ``space`` (``This Is Space Case``)
+        :param max_filename_length:
+            Minimum of maximum filename length on Windows and maximum filename
+            length pn Unix-like systems. Default value is 260 characters.
```

Comment:
typo: pn -> on
\+
No need to mention the default value, coala automatically does that.